manage/user/index.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+$root = realpath($_SERVER["DOCUMENT_ROOT"]);
+
+require_once "$root/server/user.class.php";
+$USER = new User();
+if (!$USER->hasPermission('edit_user_permission')) {
+	header("Location: /unauthorized");
+	die();
+}
+
+?>
+
+<!DOCTYPE html>
+<html class="no-js" lang="">
+<head>
+	<title>Queue Test</title>
+	<?php require_once "$root/server/header.php" ?>
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.12.4/css/bootstrap-select.min.css" />
+	<link rel="stylesheet" href="user.css" />
+</head>
+<body>
+	<?php require_once "$root/components/nav.php"; ?>
+	<div class="container">
+		<div class="row">
+			<div class="col-md-9">
+				<h1>User Management</h1>
+			</div>
+			<div class="col-md-3">
+				<button class="btn btn-default pull-right" id="btn-add-user">Add User</button>
+			</div>
+		</div>
+		<div class="row">
```

Comment:
The "row" starting on line 34 should have its overflow-x set to auto. Currently the site kinda breaks on mobile, so this is an easy solution where people can just scroll horizontally without breaking the navbar. People probably won't be using this page on mobile, but it's a simple fix to keep our site from looking bad under most circumstances.

Reply:
Done